src/pages/index.js

```diff
@@ -163,7 +163,7 @@ function handleCardClick(data) {
 /* -------------------------------------------------------------------------- */
 /*                          Form Popup : Edit Profile                         */
 /* -------------------------------------------------------------------------- */
-/* function handleProfileFormSubmit() {
+function handleProfileFormSubmit() {
   submitButtonEditProfileInfo.textContent = "Saaving...";
   // select submit button on form change text content to Saving...
```

Comment:
` name: titleInput.value, about: jobInput.value`


Only method `_getInputValues` collects all the data of the inputs and passes it to the call of `submitForm`. So you need to use the collected data rather than the inputs.